ThunderCloud/ImageSelectionCollectionViewCell.swift

```diff
@@ -35,9 +37,11 @@ class ImageSelectionCollectionViewCell: UICollectionViewCell {
     override var isSelected: Bool {
         didSet {
             guard oldValue != isSelected else { return }
-            
-            imageView.layer.borderWidth = isSelected ? 2 : 0
-            labelContainerView.backgroundColor = isSelected ? ThemeManager.shared.theme.mainColor : .clear
+
+            let tintColor = isSelected ? ImageOption.selectedColor : ImageOption.borderColor
+            imageView.layer.borderColor = tintColor.cgColor
+            imageView.layer.borderWidth = isSelected ? ImageOption.selectedBorderWidth : 1
```

Comment:
Probably unnecessary, but any use case where a `deselectedBorderWidth` (defaulting to `1` might want to be set)